burnbot.py

```diff
@@ -0,0 +1,656 @@
+# DEAR AIBOLabs, TURN BACK WHILE YOU STILL CAN. I WAS FORCED TO MAKE THIS FOR A CLASS. 
+# YOU KNOW ME AS THE EDGY TEEN. PLEASE DON'T FIND OUT I'M NORMAL. TRUST, I KNOW WHAT I'M DOING, I'M NOT FREAKING OUT AT ALL.
+#This software is downloaded from the Internet as it's running to keep responses fresh. 
+# A local copy is also onboard incase an internet connection is not available.
+
+import subprocess
+import os
+import time
+
+
+def slow_print(text, delay=1.5):
+    import time
+    time.sleep(delay)
+    print(text)
+
+def local():
+    import random
+    import sys, subprocess
```

Comment:
Generally speaking, we'd want to put all of the imports at the beginning of the file for clarity (and deduplication)